### ISSUE TITLE
Toggle Feral Gamemode when the game is running

### ISSUE THREAD

**Issue by PolpOnline** [gorilla-devs/GDLauncher#1053]
**Is your feature request related to a problem? Please describe.**
Hi, I'd like to have [Feral Gamemode](https://github.com/FeralInteractive/gamemode) automatically enabled whenever the game starts so it can disable power saving modes and maximize Minecraft's performance.
ATlauncher already does this.

**Describe the solution you'd like**
 There are plenty of ways to enable gamemode:
- Use gamemoderun to start the game ([here's its manpage](https://man.archlinux.org/man/community/gamemode/gamemoderun.1.en))
- Preload the library `libgamemodeauto.so.0` (don't know if this would work with Java) (instructions are in the same manage indicated above)
- Use `gamemoded -r $(pidof java)` to toggle game mode for Java (this is the solution I adopted in the script I made to workaround this issue. (As you may have already intuited getting Java's PID this way could be a little problematic if the user is using another Java program in the meantime)

**Comment by Eskaan** [comment]
You recommend to take the „pid of Java“. You mean the JVM running the Minecraft instance?

**Comment by PolpOnline** [comment]
Yes, that is the solution I adopted to let `gamemoded` know which process to prioritize
